pkg/bmc/system.go

```diff
@@ -7,8 +7,10 @@ package bmc
 import (
 	"crypto/ecdsa"
 	"crypto/elliptic"
+	crand "crypto/rand"
```

Comment:
I import this as crand as I know I will need math/rand for other things in other changes that are in the PR queue.